pkg/metrics/metrics.go

```diff
@@ -79,6 +102,21 @@ func (mm *metricsManager) EmitGKEComponentVersion() error {
 	return nil
 }
 
+func getErrorCode(err error) string {
+	if err == nil {
+		return codes.OK.String()
+	}
+
+	st, ok := status.FromError(err)
+	if !ok {
```

Comment:
This may not be accurate for our use case when I take a second look. 
The LoggedError func is filtering out the user error, but the RecordOperationErrorMetrics is not. We can use the existing ``func CodeForError(err error)`` and remove this getErrorCode function.